src/views/HomePage/HomePage.tsx

```diff
@@ -2,19 +2,42 @@ import { TitleBanner } from '@ui/components/TitleBanner';
 import StyledHomePage from './StyledHomePage';
 import logo from '/wwtbtfc.svg';
 import { NavigationButton } from '@ui/components/NavigationButton';
+import colors from '@ui/styles/colors';
 
 /**
  * Main home page for the app
  */
 export const HomePage = () => {
+  const oneLiners = [
```

Comment:
Since these are static values, let's use a ref so that the component doesn't have to re-declare the array on every render

```suggestion
  const oneLiners = useRef([
```